app-kotlin/src/main/kotlin/au/com/gridstone/trainingkotlin/util/NetworkConnectionInterceptor.kt

```diff
@@ -0,0 +1,54 @@
+package au.com.gridstone.trainingkotlin.util
+
+import android.content.Context
+import android.net.ConnectivityManager
+import android.net.NetworkCapabilities
+import android.os.Build
+import okhttp3.Interceptor
+import okhttp3.Response
+import java.io.IOException
+
+class NetworkConnectionInterceptor
```

Comment:
do we need this for this PR? this PR is about details screen right? 🤔

Reply:
fixed